### PR TITLE
Update table col sizes to match designs

### PR DESCRIPTION
**Pull request by radum**
## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/styles/7-components/_table.scss [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
+	&--fixed {
+		table-layout: fixed;
+	}
+
+	&__cell {
+		&--xs {
+			width: 80px;
+		}
+		&--s {
+			width: 140px;
```
Comment (radum) [inline apps/web/src/styles/7-components/_table.scss:23]: Changed this to 140 from 130 to fix the wrapping.

image

**Review by brpins — APPROVED** [review]

**Pull request merged**